### DISCUSSION TITLE
Pod test-kafka-1 will be force-rolled, due to error: null

### DISCUSSION THREAD

**Discussion by vutkin** [strimzi/strimzi-kafka-operator#5271] [General]
Hi @scholzj , could you please clarify what do this logs from strimzi-operator mean?
```
Pod test-kafka-1 will be force-rolled, due to error: null
```
Or kafka brokers starts bouncing time to time.

**Comment by vutkin** [comment]
We've found that kured is in work, but why error is null?

**Comment by scholzj** [comment]
Sorry, I do not know what it means like this. Maybe if you can post the full log to get more context it would be more clear what does it mean. This might be a bug ... since giving `null` as the reason seems a bit weird.

CC @tombentley @sknot-rh who better understand this part of the operator.

**Reply by tombentley** [reply]
Yeah we'd need more context, but I agree `null` is at best unhelpful and at worst a bug.

**Comment by sknot-rh** [comment]
@vutkin Can you provide the logs or reproducer?
